nautobot_ssot/jobs/examples.py

```diff
@@ -59,6 +61,25 @@ class SiteModel(DiffSyncModel):
     pk: Optional[UUID]
 
 
+class PrefixModel(DiffSyncModel):
+    """Shared data model representing a Prefix in either of the local or remote Nautobot instances."""
+
+    # Metadata about this model
+    _modelname = "prefix"
+    _identifiers = ("prefix", "tenant_slug")
+    # To keep this example simple, we don't include **all** attributes of a Prefix here. But you could!
+    _attributes = ("description", "status_slug")
+
+    # Data type declarations for all identifiers and attributes
+    prefix: str
+    tenant_slug: str
```

Comment:
tenant_slug can be None, right?

```suggestion
    tenant_slug: Optional[str]
```

Reply:
because I was using the prefix and tenant_slug as identifiers it can't be `None` but `""`